plugin/kubernetes/kubernetes.go

```diff
@@ -414,32 +427,34 @@ func (k *Kubernetes) findPods(r recordRequest, zone string) (pods []msg.Service,
 
 // findServices returns the services matching r from the cache.
 func (k *Kubernetes) findServices(r recordRequest, zone string) (services []msg.Service, err error) {
```

Comment:
We run with `podModeDisabled` so only this section is relevant to us, however, we should also put the changes in for `findPods`.